client/my-sites/plans-features-main/hooks/use-generate-action-hook.tsx

```diff
@@ -253,7 +254,12 @@ function useGenerateActionHook( {
 			} );
 		}
 
-		if ( isFreePlan( planSlug ) ) {
+		if ( ! canUserManageCurrentPlan && ! current ) {
+			/** Non site owners cannot intract with plans */
+			status = 'disabled';
+			text = translate( 'Contact owner' );
+			tooltipText = translate( 'Please contact the site owner to manage your plan.' );
```

Comment:
I couldn't find the link, but I'm guessing we have agreed on the wording/copy for this? e.g. having "your" in the text when you are not the site owner. I guess it's ok.

Reply:
I chose this text as a first pass since we do not have copy for the tooltip. Open to suggestions!